grails-app/views/layouts/main_component.gsp

```diff
@@ -0,0 +1,184 @@
+<%@ page import="org.broadinstitute.orsp.IssueStatus" %>
+<%@ page import="org.broadinstitute.orsp.IssueType" %>
+<%@ page import="org.broadinstitute.orsp.PreferredIrb" %>
+
+<!DOCTYPE html>
```

Comment:
This file is a straight copy of `main.gsp` with additions to support the react stuff. I envision this being where we start doing the consolidation and eventually deprecate `main.gsp` altogether.

Reply:
So in the future we'll use this as a main entry-point to all the info that the components will need?

Reply:
That is the idea, yes. We could instead use the old `main.gsp` but I wanted to try and minimize the introduction of bugs with the old code.